metabase/metabase
# Add relative date filters UI

Epic https://github.com/metabase/metabase/issues/34112

How to verify:
- Orders -> Filter -> Created At -> Relative dates
- It should be possible to set past/current/next options
- Check `Starting from` and `Include current` options

<img width="546" alt="Screenshot 2023-10-06 at 15 03 42" src="https://github.com/metabase/metabase/assets/8542534/15d99aea-a2ba-4d69-954d-262fde669eb9">
<img width="473" alt="Screenshot 2023-10-06 at 15 03 27" src="https://github.com/metabase/metabase/assets/8542534/517c85d8-8556-44b2-a467-4718a23280ea">
<img width="551" alt="Screenshot 2023-10-06 at 23 18 53" src="https://github.com/metabase/metabase/assets/8542534/487836c4-b7e8-439f-9213-90e8bf877fbf">
<img width="673" alt="Screenshot 2023-10-06 at 23 14 15" src="https://github.com/metabase/metabase/assets/8542534/508ac651-549f-49b3-8b33-71c856c64436">
